akka-actor-tests/src/test/scala/akka/actor/ActorSystemSpec.scala

```diff
@@ -115,23 +117,34 @@ object ActorSystemSpec {
 }
 
 @nowarn
-class ActorSystemSpec extends AkkaSpec(ActorSystemSpec.config) with ImplicitSender {
+class ActorSystemSpec extends AkkaSpec(ActorSystemSpec.config) with ImplicitSender with ScalaCheckPropertyChecks {
 
   import ActorSystemSpec.FastActor
 
   "An ActorSystem" must {
 
-    "reject invalid names" in {
-      for (n <- Seq(
-             "-hallowelt",
-             "_hallowelt",
-             "hallo*welt",
-             "hallo@welt",
-             "hallo#welt",
-             "hallo$welt",
-             "hallo%welt",
-             "hallo/welt")) intercept[IllegalArgumentException] {
-        ActorSystem(n)
+    "reject common invalid names" in {
+      val commonInvalidNames =
+        Table(
+          "-hallowelt",
+          "_hallowelt",
+          "hallo welt",
+          "hallo*welt",
+          "hallo@welt",
+          "hallo#welt",
+          "hallo$welt",
+          "hallo%welt",
+          "hallo/welt")
+      forAll(commonInvalidNames) { (name: String) =>
```

Comment:
What is the value of using `forAll` in this test compared to how it was? If no value then revert.

Reply:
Looks more human language, I think.

Reply:
For us, plain Scala is more natural language. 😄 
Let's revert this part.

Reply:
updated, could you help take a look